nl_open_data/flows/cbs/test_zip_csv_flow.py

```diff
@@ -0,0 +1,94 @@
+from prefect import task, Flow, unmapped, Parameter
+from prefect.tasks.shell import ShellTask
+
+from nl_open_data.utils import (
```

Comment:
this doesn't look DRY to me. You could use `inspect` to get a list of all your functions and use `eval` to create all the tasks from `statline_bq.utils`

Reply:
Trying to look into this, do you mean something like:

```
import nl_open_data.utils as stu
import inspect

nlu_functions = inspect.getmembers(nlu, inspect.isfunction)

for name, value in nlu_functions:
    name = task(value) #Doesn't actually work, since I need to solve scoping
```